pkg/sdn/plugin/node.go

```diff
@@ -207,6 +207,9 @@ func (node *OsdnNode) Start() error {
 	if err != nil {
 		return err
 	}
+	if err := node.podManager.Start(cniserver.CNIServerSocketPath); err != nil {
```

Comment:
if `go s.Serve(l)` returns an error, nothing will log/handle/restart/exit... if it panics, the process will exit. Do we need to put handling around the Serve call inside the goroutine?